tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -121,14 +120,15 @@ def get_regex_buckets(event):
         # {'regex': '.*.cmr.json$', 'sampleFileName': 'L0A_0420.cmr.json', 'bucket': 'public'}]
         regex_buckets = {}
         for regx in file_buckets:
-            regex_buckets[regx["regex"]] = buckets[regx["bucket"]]
+            regex_buckets[regx["regex"]] = buckets[regx["bucket"]["name"]]
```

Comment:
Why are we changing the input schema?

Reply:
Per @AndrewDorn  comment, I agree the fix implemented here is not correct. See my comment on the buckets dictionary and how it is created currently. The creation of the dictionary should allow for any bucket key to get the name of of the S3 bucket.